client/src/components/ItemCard/ItemCard.tsx

```diff
@@ -10,6 +12,9 @@ function ItemCard({
   cartItem: CartItem;
   updateItemQuantity: (barcode: string, quantity: number) => void;
 }) {
+  const theme = useTheme();
+  const themeSpacing = theme.spacing(1);
```

Comment:
What does theme.spacing(1) refer to?

Reply:
https://github.com/googleinterns/scan-and-go/blob/97b27fe6b1a6cc5a931f7e553380fe6910a2f8c3/client/src/components/StoreCard/StoreCard.tsx#L19-L22
Think I need to move this comment somewhere more visible...